index.js

```diff
@@ -11,40 +11,37 @@ function instrument(options) {
   var instrumenter = new Instrumenter.Instrumenter();
   var captured = false;
 
-  function transform(file) {
-    // If if doesnt match the pattern dont instrument it
+  return through.obj(function(row, enc, next) {
+    if (row.file.indexOf("node_modules") !== -1) {
```

Comment:
Yes, I think that something like this may work:
```js
var excludePattern = options.exclude ? [].concat(options.exclude) : ['**/node_modules/**'];
```